README.md

```diff
@@ -1,20 +1,22 @@
-# Fleek Non-Fungible Apps
 
-**The repository for Fleek Non-Fungible Apps project**
+# Non-Fungible Apps - by Fleek
 
 > 🚧 IMPORTANT 🚧 - This initiative is under development, so this repo should be treated as a WIP. The goals and the roadmap might change as the project is shaped.
 
 ## ⚡ Overview
 
-This is the landing point for Fleek's initiative to implement infrastructure as Solidity contracts.
+This is the landing point for our initiative to decentralize web3 app frontend infrastructure, their distribution, and access, through Non-fungible Applications, an NFT-based implementation.
 
-The vision is to have this on Ethereum Mainnet. We can create a network of smart contracts that represent the different parts of your stack. We had previously targeted Polygon but have changed it to Ethereum Mainnet for better interoperability and the security that mainnet provides.
+**The goal of NFAs** is to put web3 application frontends on-chain to provide users with a verifiable and crypto-friendly way of accessing applications.
 
-The goal is to be a more verifiable and crypto-friendly Serverless.yaml or Cloudformation manifests that will enable us to develop use cases on top like community hosting.
+1. Developers mint their app's frontend onto an NFT, where the frontend bundle is stored and referenced from IPFS.
+2. Users mint prints of the NFA, which allows them to have a decentralized and permanently on-chain access point to any web3 app.

```

Comment:
We have decided to use the term `apps` instead of `prints`

Reply:
Let's bring to broader convo, I think we are using prints elsewhere atm so for consistency I leave it - to not confuse with "NFA print, NFA app, Nonfungible application apps" can sound redundant initially.